modules/gdscript/gdscript_parser.cpp

```diff
@@ -3740,6 +3740,14 @@ bool GDScriptParser::export_annotations(const AnnotationNode *p_annotation, Node
 	// This is called after the analyzer is done finding the type, so this should be set here.
 	DataType export_type = variable->get_datatype();
 
+	if (p_annotation->name == SNAME("@export_range"))
+	{
+		if (export_type.builtin_type == Variant::INT)
+		{
+			variable->export_info.type = Variant::INT;
+		}
+	}
```

Comment:
To follow our style guidelines:

```suggestion
	if (p_annotation->name == SNAME("@export_range")) {
		if (export_type.builtin_type == Variant::INT) {
			variable->export_info.type = Variant::INT;
		}
	}
```

See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
I'll keep it mind next time, Thanks!